config/code_auto_gen.py

```diff
@@ -159,12 +159,11 @@ def convert_type_to_macro(t):
 module_bus_number = module_bus_number + '};\n#endif\n'
 
 
-
+print(global_structs)
```

Comment:
Might be good to use Jinja2 for this stuff, I know you have an issue up about the YAML but it might be nice to consolidate the autogen too

Reply:
That was the plan, I will completely re-write the autogen tools